multiqc/modules/bakta/bakta.py

```diff
@@ -0,0 +1,150 @@
+""" MultiQC module to parse output from Bakta """
+
+
+import logging
+from collections import OrderedDict
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import bargraph, table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="Bakta",
+            anchor="bakta",
+            href="https://github.com/oschwengers/bakta",
+            info="Rapid & standardized annotation of bacterial genomes, MAGs & plasmids",
+            doi="10.1099/mgen.0.000685",
+        )
+
+        # Parse logs (txt files)
+        self.bakta = dict()
+        for f in self.find_log_files("bakta"):
+            self.bakta[f["s_name"]] = self.parse_bakta(f["f"])
+            self.add_data_source(f)
+        # Filter out ignored samples (given with --ignore-samples option)
+        self.bakta = self.ignore_samples(self.bakta)
+
+        # Raise error if dict is empty
+        if len(self.bakta) == 0:
+            raise UserWarning
+
+        log.info("Found {} logs".format(len(self.bakta)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.bakta, "multiqc_bakta")
+
+        # Add most important Bakta annotation stats to general table
+        headers = OrderedDict()
+        headers["Count"] = {
+            "title": "# contigs",
+            "description": "Number of contigs",
+            "min": 0,
+            "format": "{:i}%",
```

Comment:
But it's not a percentage? Also, I don't think {:i} is a recognised format string. At least it doesn't seem to work, as numbers are displayed in the table as string (you can tell that by the lack of thousand groups separators!). I'll change them to `"{:,d}"` if that's okay.